sources/engine/Stride.Graphics/Direct3D/DXGIMwaFlags.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Stride.Graphics.Direct3D
+{
+    public enum DXGIMwaFlags : uint
+    {
+        NO_WINDOW_CHANGES = 0x1,
+        NO_ALT_ENTER = 0x2,
+        NO_PRINT_SCREEN = 0x4,
+        VALID = 0x7
+    }
```

Comment:
We'd welcome these contributions upstream in the Silk.NET library itself, our generator is still early days and doesn't necessarily generate everything today. In 3.0 we're rewriting our generator to be more stable, but that version is still in the planning stage at the moment.

Reply:
Sure! I got myself a bit more familiar with generators this week (I knew quasi nothing about it before that). I'll do my best but it might take some time!

Reply:
You can also just add these as manual files for now, which should be just a simple copy-and-paste job :) The generators are absolutely horrible to work with, so I wouldn't wish that upon a community contributor.

(and maybe changing the names a bit to fit in!)